model/task_queue.go

```diff
@@ -42,9 +42,19 @@ type DistroQueueInfo struct {
 }
 
 func GetDistroQueueInfo(distroID string) (DistroQueueInfo, error) {
+	rval, err := getDistroQueueInfoCollection(distroID, TaskQueuesCollection)
+	return rval, err
+}
+
+func GetDistroQueueInfoAlias(distroID string) (DistroQueueInfo, error) {
+	rval, err := getDistroQueueInfoCollection(distroID, TaskQueuesCollection)
```

Comment:
Should `TaskQueuesCollection` be `TaskAliasQueuesCollection`?

Reply:
```go
// ClearTaskQueue removes all tasks from task queue by updating them with a blank queue,
//and modifies the queue info.
//This is in contrast to RemoveTaskQueues, which simply deletes these documents.
```

Reply:
LGTM, mod linting nit - adding spacing between the `//` and comment text.